src/import_edf.jl

```diff
@@ -281,31 +339,40 @@ Collections of `EDF.Signal`s are mapped as channels to `Onda.Signal`s via simple
 
 `store_edf_as_onda` automatically uses a variety of default extractors derived
 from the EDF standard texts; see `src/standards.jl` and
-[`extract_channels_by_label`](@ref) for details. The caller can also provide
-additional extractors via the `custom_extractors` keyword argument, and the
+[`extract_channels_by_label`](@ref) for details. The caller can provide
+alternative extractors via the `custom_extractors` keyword argument, and the
 [`edf_signals_to_samplesinfo`](@ref) utility can be used to extract a common
 `Onda.SamplesInfo` from a collection of EDF.Signals.
 
 `EDF.Signal` labels that are converted into Onda channel names undergo the
 following transformations:
 
-- the label is whitespace-stripped, parens-stripped, and lowercased
+- the label's prepended signal type is matched against known types, if present
+- the remainder of the label is whitespace-stripped, parens-stripped, and lowercased
 - trailing generic EDF references (e.g. "ref", "ref2", etc.) are dropped
 - any instance of `+` is replaced with `_plus_` and `/` with `_over_`
 - all component names are converted to their "canonical names" when possible
   (e.g. for an EOG matched channel, "eogl", "loc", "lefteye", etc. are converted
   to "left").
 
+`EDF.Signal`s which get extracted into more than one `Onda.Samples` are removed
+and a `AmbiguousChannelError` displayed as a warning. 
```

Comment:
```suggestion
and an `AmbiguousChannelError` displayed as a warning. 
```
Also, this doesn't happen explicitly in this function, but is rather implied by the loop over `diagnostics`, which will make any exception a warning. Should it be documented that every exception becomes a warning rather than calling out this particular type? Or is this the only one that may show up here?